gutenberg.php

```diff
@@ -67,6 +67,7 @@ function the_gutenberg_project() {
  * @since 0.1.0
  */
 function gutenberg_menu() {
+	global $post;
 	if ( ! gutenberg_can_edit_post( $post ) ) {
```

Comment:
I think it doesn't make sense to keep the check actually because this menu is always rendered in the admin and not only when there's a global  $post variable (post being edited).

I think the reasoning for the PR was to hide the menu if the rich-text preference is disabled But I don't think it's very important to check for it anyway.

Reply:
I pushed a commit to remove the check entirely.